html/pfappserver/lib/pfappserver/Form/Config/Network.pm

```diff
@@ -50,14 +50,35 @@ has_field 'dhcp_max_lease_time' =>
    required_when => { 'fake_mac_enabled' => sub { $_[0] ne '1' } },
    messages => { required => 'Please specify the maximum DHCP lease time.' },
   );
+has_field 'ip_reserved' =>
+  (
+   type => 'Text',
+   label => 'IP Addresses reserved',
+   required_when => { 'fake_mac_enabled' => sub { $_[0] ne '1' } },
+   messages => { required => "Please specify the DNS server's IP address(es)." },
+   tags => { after_element => \&help,
+             help => 'Comma delimited range or IP Addresses' },
```

Comment:
A bit more explanations around what that feature does would be helpful.

It doesn't seem like its DHCP reservation, it looks more like DHCP exclusion